build/assert-publish-aot-testApp-warning-count.ps1

```diff
@@ -0,0 +1,36 @@
+Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass
+
+$trimmingWarningCount = 0
+$aotWarningCount = 0
+foreach($line in Get-Content .\build\publishAotTestAppOutput.log) 
+{
+    Write-Host $line
+    if ($line -like "*Trim analysis warning IL*") 
+    { 
+        $trimmingWarningCount += 1
+    } elseif ($line -like "*AOT analysis warning IL*") 
+    {
+        $aotWarningCount += 1
+    }
+}
+
+$expectedTrimmingWarningCount = 11
+$expectedaotWarningCount = 19
+
+Write-Host "trimmingWarningCount: ", $trimmingWarningCount
+Write-Host "aotWarningCount: ", $aotWarningCount
+
+$testPassed = 0
+if ($trimmingWarningCount -ne $expectedTrimmingWarningCount ) 
+{
+    $testPassed = 1
+    Write-Host "trimmingWarningCount: ", $trimmingWarningCount, "is not as expected. Expected count is:", $expectedTrimmingWarningCount
+}
+
+if ($aotWarningCount -ne $expectedaotWarningCount ) 
+{
+    $testPassed = 1
+    Write-Host "aotWarningCount: ", $aotWarningCount, "is not as expected. Expected count is:", $expectedaotWarningCount 

```

Comment:
```suggestion
    Write-Host "aotWarningCount: ", $aotWarningCount, " is not as expected. Expected count is: ", $expectedaotWarningCount 

```

Reply:
It looks like the commonly followed convention is to add a space after the colon:

https://github.com/open-telemetry/opentelemetry-dotnet/actions/runs/5697194997/job/15443511912?pr=4717#step:5:2306

Reply:
fixed format!